package.json

```
@@ -21,12 +21,13 @@
   },
   "homepage": "https://github.com/InkoHX/vm2-discordjs#readme",
   "dependencies": {
-    "discord.js": "^12.5.3",
+    "@discordjs/builders": "^0.12.0",
+    "discord.js": "^13.6.0",
     "vm2": "^3.9.5",
     "workerpool": "^6.1.5"
   },
   "engines": {
-    "node": ">=14.0.0"
+    "node": ">=16.9.0"
```

Discord.jsが必要とするバージョンは`v16.6.0`以上なので、最初に修正して頂いたバージョンで問題ありません。
```suggestion
    "node": ">=16.6.0"
```